inference-engine/src/low_precision_transformations/src/fake_quantize_dequantization.cpp

```diff
@@ -50,7 +51,7 @@ bool FakeQuantizeDequantization::multiplyHasZero() const {
     }
 
     auto const values = multiplyConstant->cast_vector<float>();
-    return std::any_of(values.begin(), values.end(), [](const float value) { return value == 0.f; });
+    return std::any_of(values.begin(), values.end(), [](const float value) { return (value == 0.f) || (std::abs(value) < 1.e-32); });
```

Comment:
minor: Let's handle `0/0` like `0` value. Actually it's even not handled in master because of `value == 0.f`.

Reply:
minor: This code still have to be modified to be able to pass Multiply(0) -> Add(0) cases.